src/main/webapp/WEB-INF/tags/instructor/results/questionPanel.tag

```diff
@@ -0,0 +1,84 @@
+                <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_SORTTYPE%>" value="${data.sortType}">
+                <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_SHOWSTATS%>" value="on" id="showStats-${questionPanel.question.questionNumber}">
+                <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_QUESTION_NUMBER%>" value="${questionPanel.question.questionNumber}">
+            </form>
+            <div class='display-icon pull-right'>
+                <span class="glyphicon <%= showAll && !shouldCollapsed ? "glyphicon-chevron-up" : "glyphicon-chevron-down"%> pull-right"></span>
+            </div>
+            
+        </c:if>
+        <c:choose>
+            <c:when test="${questionPanel.boldQuestionNumber}">
+                <strong>Question ${questionPanel.question.questionNumber}: </strong>
+                <div class="inline panel-heading-text">
+                    <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
+                    <span class="text-preserve-space">${questionPanel.questionText}${questionPanel.additionalInfoText}</span>
+                </div>
+            </c:when>
+            <c:otherwise>
+                Question ${questionPanel.question.questionNumber}: 
+                <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
+                <span class="text-preserve-space">${questionPanel.questionText}${questionPanel.additionalInfoText}</span>
+            </c:otherwise>
+        </c:choose>
+    </div>
+    <div <c:if test="${questionPanel.collapsible}">class="${questionPanel.responsesBodyClass}"</c:if>>
+        <div class="panel-body padding-0" <c:if test="${questionIndex != null}">id="questionBody-${questionIndex}"</c:if>>
+            
+            <c:if test="${!questionPanel.questionHasResponses}">
+                <div class="col-sm-12">
+                    <i class="text-muted">There are no responses for this question.</i>
+                </div>
+            </c:if>
+            
+            <c:if test="${showAll && questionPanel.questionHasResponses}">
+                <div class="resultStatistics">
+                    ${questionPanel.questionStatisticsTable}
+                </div>
+                <c:if test="${questionPanel.showResponseRows}">
+                    <div class="table-responsive">
+                        <table class="table table-striped table-bordered dataTable margin-0">
+                            <thead class="background-color-medium-gray text-color-gray font-weight-normal">
+                                <tr>
+                                    <c:forEach items="${questionPanel.columns}" var="thElement">
+                                        <th <c:forEach items="${thElement.attributes}" var="attr"> ${attr.key}="${attr.value}"</c:forEach>> 
```

Comment:
You can use the `getAttributesToString()` method in `ElementTag` now.
(likewise in `responseRow.tag`.)